README.md

```diff
@@ -23,3 +23,16 @@ For more information about the OpenGitOps project, see: <https://github.com/open
 ## Code of Conduct
 
 The OpenGitOps community follows the [CNCF Code of Conduct](https://github.com/open-gitops/.github/blob/main/CODE_OF_CONDUCT.md).
+
+## Reference this Content
+
+A [BibTex](https://www.bibtex.org) code snippet to use as reference for the content available in this repository is listed below, make sure to change your site visit date accordinly.
```

Comment:
in keeping with below
```suggestion
A [BibTex](https://www.bibtex.org) code snippet to use as reference for the content available in this repository is listed below, make sure to change your site visit date, and optionally release version, accordingly.
```

Reply:
fixed with https://github.com/open-gitops/documents/pull/78/commits/747bfc5836bdf76cf484a311120ee6d15c8823c1

Reply:
Just checking, you didn't want to add "and year"? For example, referencing v1.0.0 the `year` would be 2021, and v1.1.0 (assuming we get #76 out this year) would be 2024.

Reply:
That is referring to the date a person creating the research paper visit the repo and noted it as a reference on his/her bibtex bibliographical database and not the date of the release of the content. Usually link references in articles carry a date of visit in the references section at the end of the article.

Reply:
We just chatted on Slack, this was indeed a misunderstanding. I can see how you thought I was referring to the `YYYY`, but in fact I meant the [year field](https://www.bibtex.com/f/year-field/). So here's a slightly updated proposed addition (adding "year field" to be clear):

> Make sure to update the `year` field to match the release version you're referencing. See the list of releases by date [here](https://github.com/open-gitops/documents/releases).